tests/static_map/static_map_test.cu

```diff
@@ -81,25 +85,67 @@ static void generate_keys(OutputIt output_begin, OutputIt output_end)
 // User-defined key type
 // Need to specify alignment to WAR libcu++ bug where cuda::atomic fails for underaligned types:
 // https://github.com/NVIDIA/libcudacxx/issues/160
-struct alignas(8) key_pair {
-  int32_t a;
-  int32_t b;
+template <typename T>
+struct alignas(8) key_pair_type {
```

Comment:
I don't think this WAR is needed anymore.

Reply:
Removing manual alignment will cause the error below:
```
/home/yunsongw/Work/cuCollections/build/_deps/libcudacxx-src/include/cuda/std/detail/__atomic(224): error: no suitable user-defined conversion from "cuda::std::__3::__cxx_atomic_alignment_wrapper_t<key_pair_type<int32_t>>" to "key_pair_type<int32_t>" exists
...
/home/yunsongw/Work/cuCollections/build/_deps/libcudacxx-src/include/cuda/std/detail/__atomic(252): error: no operator "=" matches these operands
```